hs_core/views/user_rest_api.py

```diff
@@ -1,30 +1,53 @@
 from rest_framework.views import APIView
 from rest_framework.response import Response
 
-from theme.models import UserProfile
+from theme.forms import UserProfileForm
 
 
 class UserInfo(APIView):
     def get(self, request):
         if not request.user.is_authenticated():
             return Response({ "title": "None", "organization": "None"})
 
-        user_info = {"username": request.user.username}
+        user_info = {
+            "username": ['Username', request.user.username],
+            "email": ['Email', ''],
+            "id": ['Id',''],
+            "first_name": ['First name', ''],
+            "last_name": ['Last name', ''],
+            "middle_name": [UserProfileForm.base_fields['middle_name'].label, ''],
+            "state": [UserProfileForm.base_fields['state'].label, ''],
+            "zipcode": [UserProfileForm.base_fields['zipcode'].label, ''],
+            "country": [UserProfileForm.base_fields['country'].label, ''],
+            "date_of_birth": [UserProfileForm.base_fields['date_of_birth'].label, ''],
+            "last_four_ss": [UserProfileForm.base_fields['last_four_ss'].label, ''],
+            "phone_1": [UserProfileForm.base_fields['phone_1'].label, '']
```

Comment:
I like this trick with the forms - you can do the same thing for email/first/last using `django.contrib.auth.forms.UserChangeForm`.

Reply:
Ahhhh! I knew that was somewhere.